### PR TITLE
feat: [ZNO-2059] create a basic theme

### PR DESCRIPTION
**Pull request by ihor-romaniuk**
**Description:** create a basic theme

**Youtrack:** https://youtrack.raccoongang.com/issue/ZNO-2059

### CODE REVIEW

File: superset-frontend/src/features/home/Menu.tsx [hunk 1]
```diff
@@ -64,6 +64,7 @@ const StyledHeader = styled.header`
           ${theme.gridUnit}px
           ${theme.gridUnit * 4}px;
         max-width: ${theme.gridUnit * theme.brandIconMaxWidth}px;
+        margin-top: 2px;
```
Comment (GlugovGrGlib) [inline superset-frontend/src/features/home/Menu.tsx:67]: Please use `theme` variable here as in previous rules

Comment (GlugovGrGlib) [reply superset-frontend/src/features/home/Menu.tsx:67]: This change can be considered as an upstream pull request (up to 3 hours of internal time allowed to open PR to upstream)

**Review by GlugovGrGlib — APPROVED** [review]

**Comment by GlugovGrGlib** [comment]
You need to also fix config.py formatting before merging

File: superset-frontend/src/features/home/Menu.tsx (hunk 1 above)
Comment (ihor-romaniuk) [reply superset-frontend/src/features/home/Menu.tsx]: Changed. Thanks!

**Review by GlugovGrGlib — APPROVED** [review]

**Pull request merged**